mappings/net/minecraft/block/AbstractCandleBlock.mapping

```diff
@@ -18,3 +18,7 @@ CLASS net/minecraft/class_5540 net/minecraft/block/AbstractCandleBlock
 		ARG 1 state
 		ARG 2 world
 		ARG 3 pos
+	METHOD method_35245 isLitCandle (Lnet/minecraft/class_2680;)Z
```

Comment:
```suggestion
	METHOD method_35245 isLit (Lnet/minecraft/class_2680;)Z
```

Reply:
The method is static and specifically checks if the state is a candle *before* checking if it is lit, hence the name